hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/response/TestCleanupTableInfo.java

```diff
@@ -260,7 +260,7 @@ private void addBucketToMetaTable(OmBucketInfo bucketInfo)
    * @throws IOException if I/O error occurs in setting up data store for the
    *                     metadata manager.
    */
-  private OMMetadataManager createOMMetadataManagerSpy() throws IOException {
+  private OmMetadataManagerImpl createOMMetadataManagerSpy() throws IOException {
```

Comment:
nit: this change is no longer needed.

```suggestion
  private OMMetadataManager createOMMetadataManagerSpy() throws IOException {
```